docs/user_guide/algorithm_list.rst

```diff
@@ -139,6 +163,49 @@ Graphs are often too large to handle, so a portion of the graph is extracted. Of
     This algorithm finds a maximal subgraph that contains nodes of at least degree *k*.
 
 
+.. py:function:: subgraph.k_truss(graph: Graph(is_directed=False), k: int) -> Graph
+
+    Finds the subgraph of edges with are part of k-2 triangles.
+
+
+.. py:function:: subgraph.maximal_independent_set(graph: Graph) -> NodeSet
+
+    Finds a maximal set of independent nodes, meaning the nodes in the set share no edges with each other
+    and no additional nodes in the graph can be added which satisfy this criteria.
+
+
+.. py:function:: subgraph.subisomorphic(graph: Graph, subgraph: Graph) -> bool
+
+    Indicates whether subgraph is an isomorphic subcomponent of graph.
+
+
+.. py:function:: subgraph.sample.node_sampling(graph: Graph, p: float = 0.20) -> Graph
+
+    Returns a subgraph created by randomly sampling nodes and including edges which exist between sampled
+    nodes in the original graph.
+
+
+.. py:function:: subgraph.sample.edge_sampling(graph: Graph, p: float = 0.20) -> Graph
+
+    Returns a subgraph created by randomly sampling edges and including both node endpoints.
+
+
+.. py:function:: subgraph.sample.ties(graph: Graph, p: float = 0.20) -> Graph
+
+    Totally Induced Edge Sampling extends edge sampling by also including any edges between the nodes
+    which exist in the original graph. See the `paper <https://docs.lib.purdue.edu/cgi/viewcontent.cgi?article=2743&context=cstech>`__
+    for more details.
+
+
+.. py:function:: subgraph.sample.random_walk(graph: Graph, num_steps: Optional[int] = None, num_nodes: Optional[int] = None, num_edges: Optional[int] = None, jump_probability: int = 0.15, start_node: Optional[NodeID] = None) -> Graph
+
+    Samples the graph using a random walk. For each step, there is a jump_probability to reset the walk.
+    When resetting the walk, if the start_node is specified, it always returns to this node. Otherwise a random
+    node is chosen for each resetting. The sampling stops when any of num_steps, num_nodes, or num_edges is
+    reached.
+
```

Comment:
I think we should in some visually obvious manner differentiate the parameter names. 

I noticed that we're inconsistent in the rest of the documentation regarding how that's done. 

[`k-core`](https://metagraph.readthedocs.io/en/latest/user_guide/algorithm_list.html#subgraph.k_core) italicizes the parameter names. 

[`betweeness_centrality`](https://metagraph.readthedocs.io/en/latest/user_guide/algorithm_list.html#centrality.betweenness) shows the parameter name in a different color. 

I don't think it makes a difference which we go with, but we should be consistent in doing it.